src/fosslight_dependency/package_manager/Swift.py

```
@@ -56,14 +59,14 @@ def parse_direct_dependencies(self):
                 except Exception as e:
                     logger.warning(f'Fail to get swift dependency tree information: {e}')
             else:
-                with open(self.tmp_dep_tree_fname) as f:
+                with open(self.tmp_dep_tree_fname, encoding='utf8') as dependency_file:
                     try:
-                        deps_l = json.load(f)
+                        deps_l = json.load(dependency_file)
                         ret = self.parse_dep_tree_json(deps_l)
                     except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)